projects/Mallard/src/components/article/article-header/long-read-header.tsx

```diff
@@ -29,17 +29,13 @@ const LongReadHeader = ({
             ) : null}
             <View style={[longReadHeaderStyles.textBackground]}>
                 {kicker ? (
-                    <View style={longReadHeaderStyles.kicker}>
```

Comment:
Are there some redundant styles that are still hanging around here for `kicker` and `headline`?

Reply:
so, I wanna come back and inline all these styles and get rid of that stylesheet because we use inline styles elsewhere. This PR lazily cleans up quite a bit of them but not all. Happy to do it all here tbh?

Reply:
Whichever you'd rather but I can approve this now on that basis